hat/assets/js/apps/Iaso/utils/permissions.ts

```diff
@@ -21,6 +21,7 @@ const POLIO_SUPPLY_CHAIN_READ = 'iaso_polio_vaccine_supply_chain_read';
 const POLIO_SUPPLY_CHAIN_WRITE = 'iaso_polio_vaccine_supply_chain_write';
 const PROJECTS = 'iaso_projects';
 const REGISTRY = 'iaso_registry';
+const ORG_UNITS_CHANGE_REQUEST_APPROVE = 'iaso_org_unit_change_request_approve';
```

Comment:
I would suggest using the term "review" instead of "approve" because you can either **approve** or **reject** a change request:

```suggestion
const ORG_UNITS_CHANGE_REQUEST_REVIEW = 'iaso_org_unit_change_request_review';
```

It's not blocking. It's up to you but it could be more explicit.

Reply:
"Approve selected changes"

Reply:
I agree